options/options_helper.cc

```diff
@@ -939,7 +939,7 @@ Status GetTableFactoryFromMap(
     }
     table_factory->reset(new BlockBasedTableFactory(bbt_opt));
     return s;
-  } else if (factory_name == PlainTableFactory().Name()) {
+  } else if (factory_name == "PlainTable") {
```

Comment:
Can we have a named constant here too?

Reply:
> Can we have a named constant here too?

No, there is no such a constant defined for PlainTable.

Reply:
How about define such a constant for "PlainTable" first, then?

Reply:
> How about define such a constant for "PlainTable" first, then?

OK, done!